tests/DesugarTest.elm

```diff
@@ -119,6 +121,32 @@ desugarTest =
                         , insideModule = "A"
                         }
             }
+        , test "desugar duplicate record field" <|
+            \_ ->
+                let
+                    x : Frontend.LocatedExpr
+                    x =
+                        frontendLambda "x" "x"
```

Comment:
We probably don't need a lambda as the body - it should be irrelevant? Can we use `Frontend.Unit` instead perhaps?